docs/developer-docs/smart-contracts/advanced-features/query-stats.md

```diff
@@ -0,0 +1,62 @@
+# Query stats
+
+## Overview
+
+When users interact with a dapp, query calls may be used. Query calls cannot modify a canister's state, making it impossible to track them from the canister's code. 
+
+The **query stats** feature gives developers information about the use of each canister's query call endpoints. With this feature, an approximation of some statistics related to query stats are made available to developers as part of the existing canister status API.
```

Comment:
This makes it sound to me like we keep statistics per Endpoint, but we actually collect one set of stats per canister

Reply:
Changed it